### PR TITLE
[dom-mc] CMake in 6.0.UP04-17.08-mc

### PR DESCRIPTION
**Pull request by lucamar**
Build `CMake` release `3.9.1` (dummy) to have it listed as a visible modulefile in the `mc` software stack as well.

### CODE REVIEW

File: jenkins-builds/6.0.UP04-17.08-mc [hunk 1]
```diff
@@ -5,6 +5,7 @@
  Boost-1.65.0-CrayGNU-17.08.eb
  Charm++-6.8.0-CrayGNU-17.08.eb                     --set-default-module --hidden
  Charm++-6.8.0-CrayIntel-17.08.eb                   --hidden
+ CMake-3.9.1.eb
```
Comment (jgphpc) [inline jenkins-builds/6.0.UP04-17.08-mc]: and 6.0.UP04-17.08-gpu ?

Comment (jgphpc) [reply jenkins-builds/6.0.UP04-17.08-mc]: => https://github.com/eth-cscs/production/pull/370

Comment (jgphpc) [reply jenkins-builds/6.0.UP04-17.08-mc]: --set-default-module ?

Comment (jgphpc) [reply jenkins-builds/6.0.UP04-17.08-mc]: --set-default-module ?

Comment (lucamar) [reply jenkins-builds/6.0.UP04-17.08-mc]: I think it makes sense to use the flag `--set-default-module` if you have more than one release installed. So I believe that we can use the flag as soon as we decide to build a new release.

**Pull request merged**